tests/security/selinux/set_get_enforce.pm

```diff
@@ -0,0 +1,60 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test '# setenforce/getenforce' commands work
+#          # setenforce - modify the mode SELinux is running in
+#          #   - usage:  setenforce [ Enforcing | Permissive | 1 | 0 ]
+#          # getenforce - reports whether SELinux is enforcing, permissive, or disabled
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#105202, tc#1769801
+
+use base 'opensusebasetest';
+use power_action_utils "power_action";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use Utils::Backends 'is_pvm';
+
+sub run {
+    my ($self) = @_;
+    my $mode_old = 'Enforcing';
+    my $mode_new = 'Permissive';
+
+    $self->select_serial_terminal;
+
+    # Get the current SELinux policy mode
+    $mode_old = script_output('getenforce');
+
+    # Set to 'Permissive' mode
+    $mode_new = 'Permissive';
+    assert_script_run("setenforce $mode_new");
+    assert_script_run('setenforce 0');
```

Comment:
@lilyeyes just wondering what's the different here 
    assert_script_run("setenforce $mode_new");
    assert_script_run('setenforce 0');

Reply:
See "Summary".

Reply:
yes, I know, so the purpose is trying to set in character and also in number, right? Both are the same purpose.